src/routes/Onboarding/ImportAccount.tsx

```diff
@@ -99,10 +111,26 @@ const ImportFileTab: FC<DesktopModeProps> = ({ desktopMode, onImport }) => {
             opacity="0"
             aria-hidden="true"
             accept="application/JSON"
-            onChange={e => setFile(e.target.files[0])}
+            onChange={e => {
+              setFile(e.target.files[0])
+            }}
           />
         </Button>
-      </Box>
+        {file && (
+          <Flex ml="34px" alignSelf="center" alignItems="center">
+            <FileIcon mr="8px" />
+            <h5>{file.name}</h5>
```

Comment:
Please, think about long file name.

Reply:
@skylarrichard123 please check is current long file name view is ok